mwtext/utilities/preprocess_wikidata.py

```diff
@@ -37,7 +37,16 @@
 
 
 def preprocess_wikidata(dump, verbose=False):
+
+    def PID_comparator(claim_tuple):
+        try:
+            return sorted_PIDs.index(claim_tuple[0])
+        except:
+            return len(sorted_PIDs)+1
+
     wikidata_preprocessor = WikidataPreprocessor()
+    sorted_PIDs = wikidata_preprocessor.get_sorted_properties()
```

Comment:
I created the comparator in here just to keep things similar to `preprocess_wikitext`. The other option would be to move lines 47 and 48 out in the global scope too. What do you think?